tests/test_app.py

```diff
@@ -168,6 +210,11 @@ def test_redirects(self):
         self.assertEqual(response_7.headers["location"], "http://localhost/")
         self.assertEqual(response_8.headers["location"], "http://localhost/")
 
+        self.assertEqual(response_9.headers["location"], "http://localhost/a")
```

Comment:
How about putting them directly after the `response_12 = self.client.get("/10")`. No need to keep jumping to find out what routes these relate to?

Or test each response directly after declaring it?

Reply:
yep I'll split these out into their own tests.